src/DwitTech.AccountService.Core/Services/ActivationService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DwitTech.AccountService.Core.Interfaces;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class ActivationService : IActivationService
+    {
+        private string GetActivationCodeFromDb(string userEmail)
+        {
+            return "pttigkgkgkgkgk6666k6k6k6k6k6k6k1123ndncj";
+        }
+
+        private string GetNameFromDb(string userEmail)
+        {
+            return "Mike";
+        }
+
+        private static string GetBaseUrl()
+        {
+            return "Test.com";
+        }
+
+        private string GetActivationUrl(string userEmail)
+        {
+            string baseUrl = GetBaseUrl();
+            string activationCode = GetActivationCodeFromDb(userEmail);
+            string activationUrl = baseUrl + "/TestAppName/ActivateAccount/" + activationCode;
+            return activationUrl;
+        }
+
+        private bool SendMail(string from, string to, string subject, string body, string cc = "", string bcc = "")
+        {
+            return true;
+        }
+
+        public bool SendActivationEmail(string from, string userEmail, string templateName, string subject = "Account Activation")
+        {
+            string userActivationCode = GetActivationCodeFromDb("example@gmail.com");
```

Comment:
Why db?

Reply:
I assumed the activation code will be retrieved from the db using the method. Am I wrong?